--- gupnp-0.18.2/libgupnp/Makefile.am.linking	2012-03-19 15:05:26.307035713 +0100
+++ gupnp-0.18.2/libgupnp/Makefile.am	2012-03-19 15:05:29.892040205 +0100
@@ -114,7 +114,7 @@ if HAVE_INTROSPECTION
 GUPnP-1.0.gir: libgupnp-1.0.la
 GUPnP_1_0_gir_INCLUDES = libxml2-2.0 GObject-2.0 Soup-2.4 GSSDP-1.0
 GUPnP_1_0_gir_CFLAGS = -DGOBJECT_INTROSPECTION_SKIP $(INCLUDES) $(AM_CFLAGS)
-GUPnP_1_0_gir_LIBS = libgupnp-1.0.la
+GUPnP_1_0_gir_LIBS = libgupnp-1.0.la gmodule-2.0
 GUPnP_1_0_gir_FILES = $(introspection_sources)
 GUPnP_1_0_gir_NAMESPACE = GUPnP
 GUPnP_1_0_gir_VERSION = 1.0
